privacyidea/api/before_after.py

```diff
@@ -178,9 +178,11 @@ def before_request():
                          request.host
     # Already get some typical parameters to log
     serial = getParam(request.all_data, "serial")
-    if serial:
+    if serial and not serial == '**':
```

Comment:
It could also be *substr*, so I would do

~~~~python
if serial and not serial.startswith("*"):
~~~~